.github/workflows/linters.yml

```diff
@@ -8,25 +8,41 @@ name: Format code using black and isort
 
 on:
   push:
-    branches: [ "develop", "issue23-precommithooks" ]
-
+    branches-ignore:
+          - master
 jobs:
   linters:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Format code with black 
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
```

Comment:
python 3.10 because we known the other tools versions all run on python 3.10

Reply:
done